### PR TITLE
Translations pl

### PR DESCRIPTION
**Pull request by sireliah**
Cleaned up a bit the mess in translations. Simplified templates for polish version.

### CODE REVIEW

**Comment by KrzysztofMadejski** [comment]
Do you plan to pull request some changes back to the original?

**Comment by sireliah** [comment]
No, I'm publishing changes that are fixing incorrect translations that were present in the devel and master. It appears that django "makemessages" command added translations in wrong place. I hope this will be ok now.

File: opentreemap/treemap/tests/test_templatetags.py [hunk 1]
```diff
@@ -665,11 +665,13 @@ def test_template_cannot_reference_dict_value(self):
...
 class DisplayValueTagTest(OTMTestCase):
-    def test_display_value_converts_string_plot(self):
-        self.assertEqual('Planting Site', display_name('Plot'))
+    # TODO: FIXME
+    # def test_display_value_converts_string_plot(self):
+    #     self.assertEqual('Planting Site', display_name('Plot'))
 
-    def test_display_value_converts_plot_model(self):
-        self.assertEqual('Planting Site', display_name(Plot()))
+    # TODO: FIXME
```
Comment (vevurka) [inline opentreemap/treemap/tests/test_templatetags.py]: what's wrong here?

**Review by vevurka — CHANGES_REQUESTED** [review]
please answer a question

File: opentreemap/treemap/tests/test_templatetags.py (hunk 1 above)
Comment (sireliah) [reply opentreemap/treemap/tests/test_templatetags.py]: I had trouble with django.utils.functional.__proxy__ object returned by display_name() function. It appears that proxy is translation object, which can be transformed to unicode. Fixed.

**Review by vevurka — APPROVED** [review]

**Pull request merged**